airflow/models/dag.py

```diff
@@ -235,13 +236,21 @@ class DAG(LoggingMixin):
         'parent_dag',
         'start_date',
         'schedule_interval',
-        'full_filepath',
+        'fileloc',
         'template_searchpath',
         'last_loaded',
     }
 
     __serialized_fields: Optional[FrozenSet[str]] = None
 
+    fileloc: str
```

Comment:
This reminds me I was wondering whether `fileloc` is guaranteed to be absolute or not and had to trace a lot of code. Maybe it’s worthwhile to add this to the docstring (and the one on DagModel).